app/models/auction.rb

```diff
@@ -32,6 +33,11 @@ def lowest_bids
     bids.select { |b| b.amount == lowest_amount }.sort_by(&:created_at)
   end
 
+  def customer_name
+    return nil if customer.nil?
```

Comment:
I guess we should, although the Customer and NullCustomer model will only have one attribute